components/footer/index.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  Box,
+  Button,
+  Container,
+  Stack,
+  Text,
+  VisuallyHidden,
+  Icon
+} from '@chakra-ui/react'
+import useTranslation from 'next-translate/useTranslation'
+import { FaGithubAlt } from 'react-icons/fa'
+import { AiFillTwitterCircle } from 'react-icons/ai'
+
+interface SocialButtonProps {
+  children: React.ReactNode
+  label: string
+  href: string
+}
+
+const SocialButton: React.FC<SocialButtonProps> = ({
+  children,
+  label,
+  href
+}) => {
+  return (
+    <Button
+      rounded={'full'}
+      cursor={'pointer'}
+      as={'a'}
+      href={href}
+      target="_blank"
+      display={'inline-flex'}
+      alignItems={'center'}
+      justifyContent={'center'}
+      transition={'background 0.3s ease'}
+    >
+      <VisuallyHidden>{label}</VisuallyHidden>
+      {children}
+    </Button>
+  )
+}
+
+const Footer: React.FC = () => {
+  const { t } = useTranslation('common')
+  return (
+    <Box p={5} bottom={0} position="relative">
+      <Container
+        as={Stack}
+        maxW={'6xl'}
+        py={4}
+        direction={{ base: 'column', md: 'row' }}
+        spacing={4}
+        justify={{ base: 'center', md: 'space-between' }}
+        align={{ base: 'center', md: 'center' }}
+      >
+        <Text>{t('COPYRIGHT_LINE')}</Text>
+        <Stack direction={'row'} spacing={6}>
+          <SocialButton
+            label={'Instagram'}
+            href={'https://twitter.com/henkakuorg'}
+          >
+            <Icon as={AiFillTwitterCircle} />
+          </SocialButton>
+          <SocialButton
+            label={'github'}
+            href={'https://github.com/henkaku-center/henkaku-nengajo-frontend'}
+          >
+            <Text mr={1}>I/F</Text>
```

Comment:
Are people really familiar with I/F and SC? Myself, honestly I wouldn't know what they mean until I opened the link.

We have been talking about both repos as "Contract" and "Frontend". Why not use these words?

Reply:
The reason is that I referred to "Omise".
But, I'm not sure which is better for me.😅

And do we need this link?
I think this also needs to be discussed.

Reply:
Yes, I think it's good that you started by replicating the other project. Thank you!

I think it adds credibility to have links to the repos on the website of an open-source project. And I think they should be easy to understand.

Reply:
I understand.
I'll change the wording to "Contract" and "Frontend".

Reply:
Thank you!